airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -80,6 +80,10 @@ class KubernetesPodOperator(BaseOperator):
         /airflow/xcom/return.json in the container will also be pushed to an
         XCom when the container completes.
     :type do_xcom_push: bool
+    :param is_delete_operator_pod: If True, delete the pod after it's reached
+        a final state (independent of pod success), or the task instance gets 
+        killed. 
```

Comment:
@marengaz  it may be good to mention the default value in the doc.

Reply:
agree, it might be useful, but:
- most IDEs show the default in the help tooltip
- seems inconsistent with the rest of the current docstring and most other docstrings on the project?

Reply:
To address your question respectively:

- Explicitly mentioning default value is useful when users are reading documentation like https://airflow.apache.org/code.html
- It's not true. Please check https://airflow.apache.org/code.html?highlight=default  and search for "default". If you see docstrings without mentioning default values, it's either so obvious that you don't need to explain explicitly or the docstring was not well enough written (IMO).